lib/iris/fileformats/pp_save_rules.py

```diff
@@ -773,6 +773,21 @@ def _vertical_rules(cube, pp):
         pp.brsvd[0] = depth_coord.bounds[0, 0]
         pp.brlev = depth_coord.bounds[0, 1]
 
+    # Surface field.
+    if (
+        height_coord is None
+        and depth_coord is None
+        and pressure_coord is None
+        and soil_mln_coord is None
+        and apt_coord is None
+        and air_pres_coord is None
+        and level_height_coord is None
+        and mln_coord is None
+        and sigma_coord is None
+    ):
```

Comment:
FWIW I reckon the logic of this **_ought_** to be captured in [`iris.util.guess_coord_axis`](https://github.com/SciTools/iris/blob/933e29f120fb7d3bc7bd4813971c41475e0d3fe9/lib/iris/util.py#L231),
so that this test could be just `if not cube.coords(axis='Z'):`.

But unfortunately things like 'height' and 'depth' are _not_ recognised by guess_coord_axis, at least at present.